os/os.module.go

```
@@ -0,0 +1,284 @@
+	if ok {
+		return py.String(v), nil
+	}
+	if len(args) == 2 {
+		return args[1], nil
+	}
+	return py.None, nil
+}
+
+// getpid returns the current process id.
+func getpid(self py.Object, args py.Tuple) (py.Object, error) {
+	return py.Int(os.Getpid()), nil
+}
+
+// putenv sets the value of an environment variable named by the key.
+func putenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) != 2 {
+		return nil, py.ExceptionNewf(py.TypeError, "missing required arguments: 'key:str' and 'value:str'")
+	}
+	k, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 1), not "+args[0].Type().Name)
+	}
+	v, ok := args[1].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 2), not "+args[1].Type().Name)
+	}
+	err := os.Setenv(string(k), string(v))
+	if err != nil {
+		return nil, py.ExceptionNewf(py.OSError, "Unable to set enviroment variable")
+	}
+	return py.None, nil
+}
+
+// Unset (delete) the environment variable named key.
+func unsetenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) != 1 {
+		return nil, py.ExceptionNewf(py.TypeError, "missing one required argument: 'key:str'")
+	}
+	k, ok := args[0].(py.String)
+	if !ok {
+		return nil, py.ExceptionNewf(py.TypeError, "str expected (pos 1), not "+args[0].Type().Name)
+	}
+	err := os.Unsetenv(string(k))
+	if err != nil {
+		return nil, py.ExceptionNewf(py.OSError, "Unable to unset enviroment variable")
+	}
+	return py.None, nil
+}
+
+// os._exit() immediate program termination; unlike sys.exit(), which raises a SystemExit, this function will termninate the program immediately.
+func _exit(self py.Object, args py.Tuple) (py.Object, error) { // can never return
+	if len(args) == 0 {
+		os.Exit(0)
+	}
+	if len(args) == 1 && objectIsInt(args[0]) {
+		_ec, err := py.GetInt(args[0])
+		if err != nil {
+			os.Exit(1)
+		}
+		exit_code, err := _ec.GoInt()
+		if err != nil {
+			os.Exit(1)
+		}
```

not sure about this: it boils down to swallow silently that error. (error which would indicate some fundamental flaw in the assumptions about `int` and `int64`.)